src/content/en/fundamentals/security/credential-management/save-forms.md

```diff
@@ -63,35 +64,32 @@ you can retain the credential information while verifying its authenticity.
 
 ## Authenticate by sending a request via AJAX
 
-To authenticate the user, send a request via AJAX and respond with profile information,
-for example, in JSON format.
+To authenticate the user, deliver credential information to your server
+via an AJAX request.
```

Comment:
Maybe say fetch request instead of AJAX request. AJAX is such an old paradigm, it seems out of place when referring to a fetch() call.